gridcomps/ExtData3G/ExtDataBracket.F90

```diff
@@ -0,0 +1,116 @@
+#include "MAPL_Exceptions.h"
+#include "MAPL_ErrLog.h"
+module mapl3g_ExtDataBracket
+   use ESMF
+   use MAPL_KeywordEnforcerMod
+   use MAPL_ExceptionHandling
+   use mapl3g_ExtDataNode
+   implicit none
+   private
+
+   public :: ExtDataBracket
+
+   type ExtDataBracket
+      type(ExtDataNode) :: left_node
+      type(ExtDataNode) :: right_node
+      logical          :: disable_interpolation = .false.
+      contains
+         procedure :: get_bracket_weights 
+         procedure :: time_in_bracket
+         procedure :: set_parameters
+         procedure :: get_left_node
+         procedure :: get_right_node
+   end type ExtDataBracket
+
+contains
+
+   subroutine set_parameters(this, disable_interpolation, left_node, right_node)
+      class(ExtDataBracket), intent(inout) :: this
+      logical, intent(in) :: disable_interpolation
+      type(ExtDataNode), intent(in) :: left_node
+      type(ExtDataNode), intent(in) :: right_node
+
+      this%disable_interpolation = disable_interpolation
+      this%left_node = left_node
+      this%right_node = right_node
+   end subroutine
+
+   function time_in_bracket(this,time) result(in_bracket)
+      logical :: in_bracket
+      class(ExtDataBracket), intent(inout) :: this
+      type(ESMF_Time), intent(in) :: time
+      type(ESMF_Time) :: left_time, right_time
+      
+      left_time = this%left_node%get_interp_time()
```

Comment:
I'm going to guess that `get_interp_time()` means get the time corresponding to that node?  Not sure "interp" is the best descriptor for that.    "ref_time" seems closer, but still not ideal.